src/sentry/utils/versioning.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+
+import warnings
+from collections import namedtuple
+
+from sentry.exceptions import InvalidConfiguration
+
+
+class Version(namedtuple('Version', 'major minor patch')):
```

Comment:
This assumes everything fits within this pattern, it might be more flexible to just make this directly subclass `tuple`.